sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Generated/Models/RecoveryPointProperties.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.RecoveryServicesBackup.Models
+{
+    /// <summary> Properties of Recovery Point. </summary>
+    public partial class RecoveryPointProperties
+    {
+        /// <summary> Initializes a new instance of RecoveryPointProperties. </summary>
+        public RecoveryPointProperties()
+        {
+        }
+
+        /// <summary> Initializes a new instance of RecoveryPointProperties. </summary>
+        /// <param name="expiryTime"> Expiry time of Recovery Point in UTC. </param>
+        /// <param name="ruleName"> Rule name tagged on Recovery Point that governs life cycle. </param>
+        internal RecoveryPointProperties(string expiryTime, string ruleName)
+        {
+            ExpiryTime = expiryTime;
+            RuleName = ruleName;
+        }
+
+        /// <summary> Expiry time of Recovery Point in UTC. </summary>
+        public string ExpiryTime { get; set; }
```

Comment:
Should this be a `DateTimeOffset`, like [AzureFileShareBackupRequest.RecoveryPointExpiryTimeInUTC](https://github.com/Azure/azure-sdk-for-net/blob/4dc2e5a2fd3261e27371e0fbcae4114ce543a7a7/sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Generated/Models/AzureFileShareBackupRequest.cs#L31)?

I guess `"format": "date-time"` is missing in the [swagger definition](https://github.com/Azure/azure-rest-api-specs/blob/9eb9c45842ba2932eee57326189019adc3018397/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-01-01/bms.json#L12610-L12613). You can refer to the correct definition [here](https://github.com/Azure/azure-rest-api-specs/blob/9eb9c45842ba2932eee57326189019adc3018397/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-01-01/bms.json#L9757-L9761).

Since it's a published API, you can use autorest directives to add the missing `format` property. See guide [here](https://github.com/Azure/autorest/blob/6caa31c18ff03173a8f4a945bfe0306c94463aa8/docs/generate/directives.md).

Reply:
@archerzz , this is expected and intentional as our service will be returning a string itself.

Reply:
@archerzz : converted to DateTimeOffset as suggested